package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "private": true,
   "engines": {
-    "node": ">=6 <=8"
+    "node": ">=8.10, <=10"
```

Comment:
Do we even need to restrict it to less than 10? I'm not sure why we were doing that before.

Reply:
I think the idea was to describe which node versions we want to support. This field is never used anyway as I commented above, so maybe we can remove it entirely?